--- darknet/cfg/yolov3.cfg	2019-09-17 15:54:33.000000000 -0400
+++ yolo-obj.cfg	2019-09-18 15:19:14.000000000 -0400
@@ -1,10 +1,10 @@
 [net]
 # Testing
-batch=1
-subdivisions=1
+# batch=1
+# subdivisions=1
 # Training
-# batch=64
-# subdivisions=16
+batch=64
+subdivisions=8
 width=416
 height=416
 channels=3
@@ -17,9 +17,9 @@
 
 learning_rate=0.001
 burn_in=1000
-max_batches = 500200
+max_batches = 2000
 policy=steps
-steps=400000,450000
+steps=1600,1800
 scales=.1,.1
 
 [convolutional]
@@ -600,14 +600,14 @@
 size=1
 stride=1
 pad=1
-filters=255
+filters=18
 activation=linear
 
 
 [yolo]
 mask = 6,7,8
 anchors = 10,13,  16,30,  33,23,  30,61,  62,45,  59,119,  116,90,  156,198,  373,326
-classes=80
+classes=1
 num=9
 jitter=.3
 ignore_thresh = .7
@@ -686,14 +686,14 @@
 size=1
 stride=1
 pad=1
-filters=255
+filters=18
 activation=linear
 
 
 [yolo]
 mask = 3,4,5
 anchors = 10,13,  16,30,  33,23,  30,61,  62,45,  59,119,  116,90,  156,198,  373,326
-classes=80
+classes=1
 num=9
 jitter=.3
 ignore_thresh = .7
@@ -773,14 +773,14 @@
 size=1
 stride=1
 pad=1
-filters=255
+filters=18
 activation=linear
 
 
 [yolo]
 mask = 0,1,2
 anchors = 10,13,  16,30,  33,23,  30,61,  62,45,  59,119,  116,90,  156,198,  373,326
-classes=80
+classes=1
 num=9
 jitter=.3
 ignore_thresh = .7
